scripts/managedb/merge_default_sp_dups.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python
+# -*- coding: latin-1 -*-
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+from __future__ import division # need for seconds calculation (could be removed with Python 2.7)
+__author__ = 'fermin'
+
+from pymongo import MongoClient
+import sys
+from datetime import timedelta, datetime
+
+if len(sys.argv) != 2:
+    print "missing db name"
+    sys.exit()
+
+DB = sys.argv[1]
+COL = 'entities'
+
+need_fix = False
+
+client = MongoClient('localhost', 27017)
+db = client[DB]
+
+def warn(n, s):
+    print '%5d: WARNING: %s' % (n, s)
+    global need_fix
+    need_fix = True
+
+def duplicated_attr_name(n, list):
+    names = {}
+    for attr in list:
+        if (names.has_key(attr['name'])):
+            warn(n, 'attribute %s is duplicated' % attr['name'])
+            return True
```

Comment:
IF there is more than one attr duplicate, the script is not going to warn about more than one